src/views/viewer/DiscoverView.tsx

```diff
@@ -0,0 +1,234 @@
+  },
+  'sports': {
+    id: '5',
+    title: 'Sports',
+    icon: <SvgVideoCategoriesSports />,
+    color: '#41EE5A',
+    coverImg: 'https://eu-central-1.linodeobjects.com/atlas-assets/category-images/sports.webp',
+  },
+  'gaming': {
+    id: '7',
+    title: 'Gaming',
+    icon: <SvgVideoCategoriesGaming />,
+    color: '#9455E8',
+    coverImg: 'https://eu-central-1.linodeobjects.com/atlas-assets/category-images/gaming.webp',
+  },
+  'comedy': {
+    id: '9',
+    title: 'Comedy',
+    icon: <SvgVideoCategoriesComedy />,
+    color: '#4FE1F2',
+    coverImg: 'https://eu-central-1.linodeobjects.com/atlas-assets/category-images/comedy.webp',
+  },
+  'autos-and-vehicles': {
+    id: '2',
+    title: 'Autos & Vehicles',
+    icon: <SvgVideoCategoriesAutosAndVehicles />,
+    color: '#6E5FEC',
+    coverImg: 'https://eu-central-1.linodeobjects.com/atlas-assets/category-images/autos-and-vehicles.webp',
+  },
+  'howto-and-style': {
+    id: '12',
+    title: 'Howto & Style',
+    icon: <SvgVideoCategoriesHowtoAndStyle />,
+    color: '#E57827',
+    coverImg: 'https://eu-central-1.linodeobjects.com/atlas-assets/category-images/howto-and-style.webp',
+  },
+  'music': {
+    id: '3',
+    title: 'Music',
+    icon: <SvgVideoCategoriesMusic />,
+    color: '#6EEC3A',
+    coverImg: 'https://eu-central-1.linodeobjects.com/atlas-assets/category-images/music.webp',
+  },
+  'nonprofits-and-activism': {
+    id: '15',
+    title: 'Nonprofits & Activism',
+    icon: <SvgVideoCategoriesNonprofitsAndActivism />,
+    color: '#E141D6',
+    coverImg: 'https://eu-central-1.linodeobjects.com/atlas-assets/category-images/nonprofit-and-activism.webp',
+  },
+  'news-and-politics': {
+    id: '11',
+    title: 'News & Politics',
+    icon: <SvgVideoCategoriesNewsAndPolitics />,
+    color: '#48F0B3',
+    coverImg: 'https://eu-central-1.linodeobjects.com/atlas-assets/category-images/news-and-politics.webp',
+  },
+} as const
+const featuredVideoCategories: Array<{ videoUrl: string; videoTitle: string } & VideoCategoryData> = [
+  {
+    ...videoCategories['travel-and-events'],
+    videoUrl: 'https://eu-central-1.linodeobjects.com/atlas-assets/featured-videos/5778.mp4',
+    videoTitle: 'Wreck Diving. Philippines, Boracay.',
+  },
+  {
+    ...videoCategories['education'],
+    videoUrl: 'https://eu-central-1.linodeobjects.com/atlas-assets/featured-videos/3941.mp4',
+    videoTitle: 'ФИЛОСОФИЯ СТОИЦИЗМА',
+  },
+  {
+    ...videoCategories['film-and-animation'],
+    videoUrl: 'https://eu-central-1.linodeobjects.com/atlas-assets/featured-videos/5984.mp4',
+    videoTitle: 'taxi driver edit',
+  },
+]
+
+export const DiscoverView: React.FC = () => {
+  const { videosConnection } = useVideosConnection({
+    first: 0,
+  })
+  const isMdBreakpoint = useMediaMatch('md')
+  return (
+    <StyledLimitedWidthContainer big>
+      <Text variant="h2">Discover</Text>
+      <FeaturedCategoriesContainer>
```

Comment:
Why aren't we using our `LayoutGrid` component for this and `CategoriesContainer`?

Reply:
now that I think of it it could very well be used, it just still feels very awkward to have it set to 12 fixed columns and then doing the conversion to what the actual columns that I want.

Reply:
I'm not sure why this is awkward as you can easily see the number of required columns in the design. Let's try to follow the design as closely as possible and use columns grid